packages.md

```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
 
 If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
 
+## Configuration
+
+- [github.com/kelseyhightower/envconfig](https://github.com/kelseyhightower/envconfig) - get configs from environment variables
+
 ## Database
 
 - [database/sql](https://golang.org/pkg/database/sql/) - standard SQL library that you should stick to or build upon
-- https://github.com/jmoiron/sqlx - extension to standard SQL library that makes it a bit more convenient
+- [github.com/jmoiron/sqlx](https://github.com/jmoiron/sqlx) - extension to standard SQL library that makes it a bit more convenient
+- [github.com/mattes/migrate](https://github.com/mattes/migrate) - database migrations
+
+## Forms
+
+- [github.com/gorilla/schema](https://github.com/gorilla/schema) - decode forms and urls into Go structs and vice versa
+
+## HTTP
+
+- [github.com/go-chi/chi](https://github.com/go-chi/chi) - router with decent performance and middleware support
+- [github.com/dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) - JSON Web Token library
+- [github.com/hellofresh/health-go](https://github.com/hellofresh/health-go) - adds health status check endpoints to your app
+- [github.com/rafaeljesus/retry-go](https://github.com/rafaeljesus/retry-go) - retry mechanism to build into your HTTP clients
+- [github.com/sony/gobreaker](https://github.com/sony/gobreaker) - CircuitBreaker implementation, use this in your HTTP clients
+
+
+## Logging and Metrics
+
+- [github.com/sirupsen/logrus](https://github.com/sirupsen/logrus) - logging package
+- [github.com/hellofresh/logging-go](https://github.com/hellofresh/logging-go) - HelloFresh extensions to Logrus
+- [github.com/hellofresh/stats-go](https://github.com/hellofresh/stats-go) - collect and push metrics
+
```

Comment:
Please add `github.com/hellofresh/gcloud-opentracing` and `github.com/opentracing/opentracing-go` - tools that we use for distributed tracing.